mav_trajectory_generation/src/segment.cpp

```diff
@@ -247,4 +247,18 @@ bool Segment::getSegmentWithAppendedDimension(const Segment& segment_to_append,
   return true;
 }
 
+bool Segment::offsetSegment(const Eigen::VectorXd& A_r_B) {
```

Comment:
nice!
Would it make sense to extend this to rotations as well? Sort of assembling the 3d position of the control points, transform them and push back the offsets to the corresponding polynomial? (maybe for future stuff)

BTW, a trajectory transformation for the sampled trajectory waits in PR https://github.com/ethz-asl/mav_comm/pull/77

Reply:
Future stuff ;)

Offsetting was easy and enough to transform between different ENU frames. Rotation probably requires a change of basis.